tools/fuchsia/fidl/BUILD.gn

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Fuchsia Authors. All rights reserved.
```

Comment:
this copy right message is inconsistent with the rest of the repo, it has to be:

```
// Copyright 2013 The Flutter Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
```

Could you please change every copyright in this PR to be the above?

Reply:
Done.